src/python/WMComponent/PhEDExInjector/PhEDExInjectorPoller.py

```diff
@@ -85,6 +86,7 @@ def setup(self, parameters):
                                 dbinterface = myThread.dbi)
         self.setStatus = daofactory(classname = "DBSBufferFiles.SetPhEDExStatus")
         self.setBlockClosed = daofactory(classname = "SetBlockClosed")
+        self.getSubscribionInfoByDataset = daofactory(classname = "GetSubscribionInfoByDataset")
```

Comment:
I think you want to use the PhEDEx DAOFactory (above), no? And there is a typo in the class name